### ISSUE TITLE
On opening TEAM page nothing is opening

### ISSUE THREAD

**Issue by fatma-sia27** [garg3133/JagratiWebApp#218]
Hey @garg3133 , When I opened TEAM section then nothing is opening it is showing the home page only , so if you like then I can make a TEAM page for your website . Please allow me to work on this .
Thank you

**Comment by garg3133** [comment]
Yeah, that'll be good @fatma-sia27. Do ahead, assigning you the issue.

Basically, we need the web team and major open source contributors on that page.

**Comment by fatma-sia27** [comment]
> Yeah, that'll be good @fatma-sia27. Do ahead, assigning you the issue.
> 
> Basically, we need the web team and major open source contributors on that page.

Hey @garg3133 , I actually want to add images and names of your team in the team page . So you should provide the information so that I can make it.
Thank you

**Comment by garg3133** [comment]
@fatma-sia27 You can make the team page with some dummy photos and names for now, we'll update it ourselves later.

**Comment by fatma-sia27** [comment]
ok ok @garg3133 I will definitely do that . But can you please let me know  I should add the positions like web developers, volunteers etc. or I should make it without their positions? And I should add then you have to tell me the different positions I should make.

**Comment by garg3133** [comment]
@fatma-sia27 You can add 4 positions, "Patrons", "Volunteers", "Web Team" and "Open Source Contributors".

**Comment by fatma-sia27** [comment]
> @fatma-sia27 You can add 4 positions, "Patrons", "Volunteers", "Web Team" and "Open Source Contributors".

ok ok thanks

**Comment by fatma-sia27** [comment]
> Hello @fatma-sia27, you claimed this issue to work on it, but this issue and any referenced pull requests haven't been updated for 10 days. Are you still working on this issue?
> 
> If so, please update this issue by leaving a comment on this issue to let me know that you're still working on it. Otherwise, I'll automatically remove you from this issue in 4 days.
> 
> If you've decided to work on something else, simply unassign yourself from this issue by going to the `Assignees` section and clicking on `unassign me`, so that someone else can claim it and continue from where you left off.
> 
> Thank you for your valuable contributions to JagratiWebApp!
Yes Iam working on it .

**Comment by fatma-sia27** [comment]
Hey @garg3133 , meanwhile can you tell me the steps to make PR as I am a beginner.

**Comment by garg3133** [comment]
@fatma-sia27 You can checkout this repository: https://github.com/Bitbyte-TPC/first-bit. It contains details explanation of every step on how to use clone a project, create branch, make changes and then push your code and create a pull request.

Always remember to  create a new branch before committing and pushing the code to your GitHub account.

**Comment by fatma-sia27** [comment]
> @fatma-sia27 You can checkout this repository: https://github.com/Bitbyte-TPC/first-bit. It contains details explanation of every step on how to use clone a project, create branch, make changes and then push your code and create a pull request.
> 
> Always remember to create a new branch before committing and pushing the code to your GitHub account.

Ook thanks

**Comment by fatma-sia27** [comment]
Hey @garg3133 , I had completed my work. Can you please tell me that in which folder should I add my team page?
![WhatsApp Image 2021-04-25 at 10 21 51 AM](https://user-images.githubusercontent.com/77666526/115981330-88083b00-a5b0-11eb-9968-d57e7dc9af6d.jpeg)

**Comment by garg3133** [comment]
@fatma-sia27 Put it inside the home directory.

**Comment by fatma-sia27** [comment]
> @fatma-sia27 Put it inside the home directory.

Ook

**Comment by fatma-sia27** [comment]
Hey @garg3133 , I had created PR#256. Please review it and let me know if any changes required.
Thank you

**Comment by fatma-sia27** [comment]
@garg3133 please review my PR#256

**Comment by fatma-sia27** [comment]
> Hello @fatma-sia27, you claimed this issue to work on it, but this issue and any referenced pull requests haven't been updated for 10 days. Are you still working on this issue?
> 
> If so, please update this issue by leaving a comment on this issue to let me know that you're still working on it. Otherwise, I'll automatically remove you from this issue in 4 days.
> 
> If you've decided to work on something else, simply unassign yourself from this issue by going to the `Assignees` section and clicking on `unassign me`, so that someone else can claim it and continue from where you left off.
> 
> Thank you for your valuable contributions to JagratiWebApp!

I had created PR#256 pls review it

**Comment by garg3133** [comment]
Linked your PR to this issue.
